### ISSUE TITLE
A11y_AzureDataStudioAAE_Sql Instance 1_Connection Strings_Zoom: Data of the "ADO.NET" edit field is not completely visible on the screen, when the screen is zoomed to 200%

### ISSUE THREAD

**Issue by krrish2021** [microsoft/azuredatastudio#15755]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Zoom;#WCAG1.4.4;#AINotInScope;

### Prerequisite:

- Set the text size to 200% in "Ease of access settings"

### Environment Details:
Application Name: Azure Data Studio -AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till Azure Arc Controller tree and tab to the node present under it
4. Tab to child node "Sqlinstance1" and open manage and tab to "Connection Strings" page
5. Tab to "ADO.NET" edit field and check if data inside it is completely visible or not.

### Actual:
Data of the "ADO.NET" edit field is not completely visible on the screen, when the screen is zoomed to 200%
similar issue is observed in the scenario given below:

1. PG1_Overview(Dashboard)
2. PG1_Properties
3. PG1_Connection Strings
4. PG1_Coordinator Node Parameter
5. PG1_Worker Node Parameter

### Expected:
Data of the "ADO.NET" edit field should completely visible on the screen, when the screen is zoomed to 200% or if not visible then tool tip should be provided to it with complete data.

### User Impact:
Keyboard only users will face difficulty in getting the information.

**Comment by Charles-Gagnon** [comment]
@krrish2021 I'm closing this as working as intended - contents of edit fields like this can be viewed by using the arrows keys to scroll the content within the field

![zoom](https://user-images.githubusercontent.com/28519865/127541254-31f189f7-b433-4540-8c87-9cf808e56660.gif)

Let us know if this isn't acceptable behavior to meet the guidelines and we can look into other options.

**Comment by krrish2021** [comment]
Closing as not repro

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
